src/SMU-Keithley_4200-SCS/main.py

```diff
@@ -236,8 +235,13 @@ def connect(self):
                 # not supported yet with pylptlib -> # self.inst = inst
                 self.param = param
 
-            ret = self.lpt.initialize()
-            
+            try:
+                ret = self.lpt.initialize()
+            except ConnectionRefusedError as e:
+                debug("Unable to connect to a lptlib server application running on the 4200-SCS. Please check your"
+                      "network settings and make sure the server application is running.")
+                raise e
```

Comment:
```suggestion
                raise
```
In the except: block a bare `raise` already raises the currently handled exception. This also makes the traceback clearer as it does not contain the rather confusing `raise e in line 243`